yb-voyager/cmd/templates/assessmentReport.template

```diff
@@ -176,6 +186,17 @@
         {{if not $hasUnsupportedFeatures}} <!-- Check if no unsupported features were found -->
             <p>No unsupported features were detected among the ones assessed.</p>
         {{end}}
+
+        <div class="notes-box">
+            <h4>Notes</h4>
+            {{if .Notes}}
```

Comment:
let's have the if condition at the div level. 
I.e. if Notes is empty, let's not even have the div, heading, etc.

Reply:
changes done, @makalaaneesh